RELEASE_NOTES.md

```diff
@@ -5,6 +5,7 @@
 - fixed: package version retrieval outside of git repository
 - fixed: checking whether a branch is merged to parent works for branches that have no common commit
 - added: CI/CD check ensuring that RELEASE_NOTES are up to date
+- added: fetching multiple PRs at once
```

Comment:
Pls specify what subcommands have been added &mdash; see e.g. the below release notes for 3.3.0 or 3.2.0 (`added: ` parts)